core/templates/dev/head/pages/exploration_editor/translation_tab/AudioTranslationBarDirective.js

```diff
@@ -93,6 +93,13 @@ oppia.directive('audioTranslationBar', [
           $scope.checkingMicrophonePermission = false;
           $scope.audioTimerIsShown = true;
           $scope.audioIsCurrentlyBeingSaved = false;
+          // 82 belongs to the keycode for 'R'
```

Comment:
Move this below, just above the if statement. Also, might be better to use https://www.w3schools.com/jsref/jsref_charcodeat.asp and delete the comment.

Just a question from the UX perspective, do you want to have the shortcut as capital R or would small r do?

Reply:
It works on pressing both r and R. It is not case sensitive